bigbluebutton-html5/imports/startup/server/userPermissions.js

```diff
@@ -128,17 +128,14 @@ export function isAllowedTo(action, credentials) {
     userId,
   });
 
-  const allowedToInitiateRequest =
-    user &&
+  const allowedToInitiateRequest = user &&
     user.authToken === authToken &&
     user.validated &&
     user.clientType === 'HTML5' &&
     user.user &&
     user.user.connection_status === 'online';
 
-  const listOfSafeActions = [
-    'logoutSelf',
-  ];
+  const listOfSafeActions = new Array('logoutSelf');
```

Comment:
I used the new operator because it was the only way i could get the linter not to complain about the syntax.